docs/src/local_development.md

```diff
@@ -0,0 +1,46 @@
+# Local Development Instructions
+
+To install the `ethstaker-deposit-cli`, follow these steps:
+
+## Prerequisites
+
+Ensure you have the following software installed on your system:
+
+- **Git**: Version control system to clone the repository. [Download Git](https://git-scm.com/downloads)
+- **Python 3.12+**: The programming language required to run the tool. [Download Python](https://www.python.org/downloads/)
+- **pip**: Package installer for Python, which is included with Python 3.12+.
+
+
+## Local Development Steps
+
+1. **Clone the Repository**
+
+    ```sh
+    git clone https://github.com/eth-educators/ethstaker-deposit-cli.git
+    ```
+
+2. **Navigate to the Project Directory**
+
+    ```sh
+    cd ethstaker-deposit-cli
+    ```
+
+3. **Install Dependencies**
+
+    ```sh
+    pip3 install -r requirements.txt
```

Comment:
Running everything under a virtual environment is probably a good practice for a Python project like this one on a local development setup. I would add a step before the dependencies install to create and activate the venv. There are a bunch of instructions throughout the project about this and they should probably be rewritten too.

Reply:
Added a section and mentioned it is optional.